src/components/compound/DlTabs/components/DlTab.vue

```diff
@@ -50,7 +51,8 @@ export default defineComponent({
         tooltip: { type: String, default: null },
         tabindex: { type: String, default: '0' },
         fontSize: { type: String, default: '18px' },
-        bordered: { type: Boolean, default: true }
+        bordered: { type: Boolean, default: true },
+        icon: { type: String, default: '' }
```

Comment:
default of null would solve the issue of checking '' empty icon. 
and if you want to guarentee no one passes empty string then just check it in a validator

Reply:
@fadiDL I prefer avoiding default null for strings, because it can mess up the types.
 I see it's widely used. What do you think ?

Reply:
@guyDataloop null is a more valid approach due to the fact that it cannot cause vue errors and it indicates that the prop does not exist. 
having a value on the prop means it exists and should be acknowledged. thats why we use null when the prop is optional and does not have a default value